packages/core/src/concurrent/concurrency-token-provider.ts

```diff
@@ -24,6 +24,10 @@ export class ConcurrencyTokenProvider implements Disposable {
       this.concurrencyCheckers = Math.max(Math.ceil(concurrency / 2), 1);
       this.checkerToken$ = range(this.concurrencyCheckers);
       this.concurrencyTestRunners = Math.max(Math.floor(concurrency / 2), 1);
+
+      this.concurrencyCheckers = 5;
```

Comment:
Let op dat je dit uiteindelijk niet vast instelt. Voor demo doeleinden is dit prima